lib/pbench/common/configtools.py

```diff
@@ -34,9 +35,11 @@ def file_list(root):
     rootdir = os.path.dirname(root)
     flist = [root]
     dirlist = [
-        os.path.abspath("%s/%s" % (rootdir, x))
-        if not os.path.isabs(x)
-        else os.path.abspath(x)
+        (
+            os.path.abspath("%s/%s" % (rootdir, x))
+            if not os.path.isabs(x)
+            else os.path.abspath(x)
+        )
         for x in dirlist
     ]
```

Comment:
Rude.  😛

Reply:
Actually, despite the extra level of indentation, I think I kinda like the idea of parenthesizing and indenting the broken ternary ... it seems to set it off a bit more clearly, visually. (Though I can't imagine deliberately *writing* it that way, I'm not sure I mind `black` doing it 🤔 )